### PR TITLE
add PR vote template

### PR DESCRIPTION
**Pull request by CJ-Wright**
@conda-forge/core 
This PR falls under `Modifying the governance document`, please vote and/or comment on this PR. 
This PR needs (75% + 1) of core to vote yea to pass. 
To vote please leave Approve (yea) or Request Changes (nay) reviews.
If you would like changes to the current language please leave a comment or push to this branch. 
This vote will end on August 22nd 2018.

### CODE REVIEW

File: src/governance.rst [hunk 1]
```diff
@@ -84,6 +84,16 @@ retain anonymity.
...
 
+To call for a standard vote, here is a template PR comment:
+```
+@conda-forge/core 
+This PR falls under {policy}, please vote and/or comment on this PR. 
+This PR needs {policy_percent} of core to vote yea to pass. 
+To vote please leave Approve (yea) or Request Changes (nay) reviews.
+If you would like changes to the current language please leave a comment or push to this branch. 
+This vote will end on {date}.
+```
```
Comment (scopatz) [inline src/governance.rst]: This is rST, not markdown

**Comment by scopatz** [comment]
Thanks @CJ-Wright - Text looks good, but the format is wrong.

**Comment by CJ-Wright** [comment]
@conda-forge/core 
This vote ends tomorrow. If you haven't voted please do. This needs 13 yeas to pass.

**Comment by scopatz** [comment]
This needs one more to vote to pass

**Comment by scopatz** [comment]
Thanks all!

**Pull request merged**